src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -544,25 +549,119 @@ void AddRuntimeLibraries (ZipArchiveEx apk, string [] supportedAbis)
 			}
 		}
 
+		bool IsWrapperScript (string path, string link)
+		{
+			if (Path.DirectorySeparatorChar == '/') {
```

Comment:
How/why would `path` contain `\` when running on Linux/macOS? That feels like something that shouldn't happen?

Reply:
The answer is on the line of your previous comment :)

```xml
<Link>lib\arm64-v8a\wrap.sh</Link>
```

This will be passed to the task as-is and since our convention is to use Windows path separator character in MSBuild files, this is entirely expected.

Reply:
mono's `msbuild` "normalizes" paths.  `dotnet msbuild`, on the other hand, does not. :-(

Consider `x.targets`:

```xml
<?xml version="1.0" encoding="utf-8"?>
<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <ItemGroup>
    <Foo Include="item1">
      <Link>path\to\whatever</Link>
    </Foo>
  </ItemGroup>
  <Target Name="Foo">
    <Message Text="%(Foo.Link)" />
  </Target>
</Project>
```

With Mono's `msbuild`, it's normalized:

```shell
$ msbuild /nologo /v:m x.targets
  path/to/whatever
```

It's *not* normalized with `dotnet msbuild`:

```
% dotnet msbuild /nologo /v:m x.targets
  path\to\whatever
```

Reply:
I'm not sure I want to rely on this behavior... There's no guarantee msbuild on *nix won't revert to the dotnet msbuild way, plus, what happens when we switch to net5/6? I'd rather be on the safe side and keep this code